querydsl-jdo/pom.xml

```diff
@@ -146,6 +146,7 @@
         </executions>
         <configuration>
           <archive>
+            <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
```

Comment:
Are manifest entries (below) still added if a manifest file is provided?

Reply:
yes, you can check that final jar contains also Automatic-Module-Name. These missing OSGI headers can be fixed also that we change <packaging> from jar to bundle. (bundle also create a jar archive)